avail-subxt/Cargo.toml

```diff
@@ -35,6 +35,7 @@ serde-hex = "0.1"
 scale-info = { version = "2.5.0", features = ["bit-vec"] }
 
 
+
```

Comment:
This change is dangerous. I cannot allow it!

Reply:
oops! sorry boss for that change! reverting